### DISCUSSION TITLE
[Feature] Change order of journal, notes and tasks

### DISCUSSION THREAD

**Discussion by MaDill** [TechbeeAT/jtxBoard#1043] [Ideas]
**Describe the solution you'd like**
I want to change the order of journal, notes and tasks. I know I can disable them, but for me the most important is tasks, then notes and last journal. I prefer a setting where I can set the order by my self.

**Comment by patrickunterwegs** [comment]
Hi @MaDill , 
thanks for your suggestion. Basically the last used tab is already saved, so every time you open the app, the last used tab comes up again. I think this really covers most needs, changing the order of the tabs is really something I don't see necessary. I'll add this to discussions to give others the chance to endorse the idea.
